### ISSUE TITLE
Prepare apps for sharing

### ISSUE THREAD

**Issue by stevehadd** [informatics-lab/forest#17]
Having whetted the appetite of the scientists and meteorologists of the WCSSP SE Asia project with a demo of the new "whiz-bang tool"  (quote from actual scientist),  we would like to invite them to try it out and give us feedback, both on the tool and the model performance.  Before we can do that, there are some issues to deal with:
* users affecting each others plots (separate issue #1)
* scaling -  need to consider how many users will be able to access the site at a time 
* URL - would be nice to create a human-readable URL, which also won't change if the machine IP changes.

**Comment by stevehadd** [comment]
We've made some good progress on this issue this week:
* app state hopefully no longer shared
* permanent URL (http://forest.informaticslab.co.uk:8888)

we show come up with a list of things that we want to have done before we start sharing the link and encouraging other to try it out. Here is a proposed list:
- [ ] implement resource management so that resource available will scale with users (MET #17)
- [ ] add a colorbar to plots (MET #7 )
- [ ] increase resolution of plots (MET #22)
- [ ] improve stability through code rationalisation (MET #20 )
- [ ] implement some feedback so user sees something while data is loading (MET  #14)

I have create a new milestone "early user testing" for these issues.

**Comment by stevehadd** [comment]
Server has now been upgraded to kubernetes. I have also added config for a dev server. Server is now ready for general use (though still quite slow).
